resources/views/admin/edit-user.blade.php

```diff
@@ -1,248 +1,125 @@
-                        <input type="password" class="form-control" id="password" autocomplete="new-password" name="password" placeholder="Password" data-lpignore="true">
-                    </div>
-                    <div class="form-group password-container col-xs-10 col-sm-4 col-md-12 col-lg-12">
-                        <input type="password" class="form-control" id="password-confirm" autocomplete="new-password" name="password_confirmation" placeholder="Confirm Password" data-lpignore="true">
-                    </div>
-                </div>
-                <div class="checkbox col-xs-10 col-sm-4 col-md-12 col-lg-12">
-                    <label>
-                        <input type="checkbox" id="active" name="active"
-                        @if($user->active)
-                        checked
-                        @endif
-                        />&nbsp;Is Active?
-                    </label>
-                </div>
-                <div class="checkbox col-xs-10 col-sm-4 col-md-12 col-lg-12">
-                    <label>
-                        <input type="checkbox" id="email_verified" name="email_verified"
-                               @if($user->email_verified)
-                               checked
-                               @endif
-                        />&nbsp;Email Verified?
-                    </label>
-                </div>
-                <div class="col-xs-10 col-sm-4 col-md-12 col-lg-12" style="padding-bottom: 20px">
-                    <label for="spam-type">Spam Type</label>
-                    <input type="text" readonly class="form-control" id="spam-type" name="spam-type" data-lpignore="true" value="{!! $user->spam_type !!}">
-                </div>
-                <button style="margin-left: 15px;" type="submit" class="btn btn-default btn-lg btn-primary">Save</button>
-                <input type="hidden" name="id" id="id" value="{!! $user->id !!}"/>
-            </form>
-    </div>
-@stop
+@append
+@section('meta')
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+@append
+@section('css')
+    {!! HTML::style('assets/css/editUser.css') !!}
+@append
 @section('scripts')
-    <!-- include summernote css/js -->
-    <link href="//cdnjs.cloudflare.com/ajax/libs/summernote/0.8.12/summernote.css" rel="stylesheet">
-    <script src="//cdnjs.cloudflare.com/ajax/libs/summernote/0.8.12/summernote.js"></script>
-
-    <script type="application/javascript">
-
-        var urls = {
-            fetchGroups: '{!!URL::action("Api\GroupApiController@getAll")!!}',
-        };
+    <script>
+        const current_groups = [];
+        @foreach($user->getGroups() as $group)
+        current_groups.push({"id": {!!$group->id!!}, "name": "{!!$group->name!!}"});
+        @endforeach
 
-        var current_language = '{!!$user->language!!}';
-        var current_country  = '{!!$user->country_iso_code!!}';
-        var current_gender   = '{!!$user->gender !!}';
+        const initialValues = {
```

Comment:
could we create a serializer for this to avoid this 
and serialize from controller method?

Reply:
done